{{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/templates/base.html

```diff
@@ -59,6 +59,24 @@
     {% block css %}{% endblock %}
 
     {% include 'base/_analytics.html' %}
+
+    {# Opbeat JS error tracking #}
+    {% if settings.OPBEAT %}
+      <script src="https://d3tvtfb6518e3e.cloudfront.net/3/opbeat.min.js"
+          data-org-id="{{ settings.OPBEAT.ORGANIZATION_ID }}"
+          data-app-id="{{ settings.OPBEAT.APP_ID }}">
+      </script>
+
+      {% if user.is_authenticated() %}
+        <script>
+        _opbeat('setUserContext', {
+            email: "{{ user.email }}",
```

Comment:
I think we'll need to remove this line, it's more project specific. You can leave the ID one though.